### DISCUSSION TITLE
Is there any way to restore the "Copy to Clipboard", "Upload" and "Save" buttons?

### DISCUSSION THREAD

**Discussion by NathanPB** [flameshot-org/flameshot#2440] [Q&A] [answered]
I noticed that those were replaced with a single "Accept the capture" button, imo it was a UX downgrade. I would really like to restore those other buttons, is there any way I could do this without having to adjust and recompile the software by myself?

Or am I just being stupid and missing some point?

Thanks for everyones attention

**Comment by borgmanJeremy** [comment, marked as answer]
How are you starting flameshot? If you start it with no "final actions" then all the buttons  be there.

Final actions are things like pin, copy, save, upload.

When you start flameshot with final actions you have already told flameshot how you want to exit, so there is just an accept button.

We are not going back to the old behavior because it was very fragile, bug prone, and poorly defined.

**Reply by NathanPB** [reply]
understandable, this does quite of what I need. Is there any way to specify the default path for the "Save" button? If I use ``-p [path]`` in the CLI the old buttons are vanished again, but if I don't them the "Save" button will prompt the path to save the file (I'd like to specify this path via CLI)

**Reply by mmahmoudian** [reply]
@NathanPB you can put the in your config (either the graphical configuration, or by editing the config file manually)

**Comment by mmahmoudian** [comment]
Read this short text about final actions:
https://flameshot.org/docs/advanced/commandline-options/#final-actions
